### ISSUE TITLE
Adding Alt Text & Keyboard Control

### ISSUE THREAD

**Issue by mgifford** [openlayers/ol2#379]
I think we can add a lot of accessibility improvement by adding support for tabbing through the interface, including alt text to the images.

I don't think this is going to be a big change.  I think I've done it to these files

lib/OpenLayers/Control/OverviewMap.js
lib/OpenLayers/Control/PanZoom.js
lib/OpenLayers/Renderer/Canvas.js

in my fork:

https://github.com/mgifford/openlayers

I'm largely a PHP developer though so I'm sure I've missed stuff.  

Anyways, it would be great to add WAI-ARIA support, but at the very least we should be able to add alt tags & keyboard only support I would hope.

**Comment by alibama** [comment]
hi mgifford - is this something that's been added in to openlayers formally?  I'm trying to get this done on https://www.cvillecouncil.us (the little map on the left) and having no luck finding documentation for OpenLayers.Control.PanPanel alt tags... thanks!

**Comment by mgifford** [comment]
I don't think it has been added. Best to check the examples from the latest release http://openlayers.org/en/v3.2.1/examples/

Worth doing!

**Comment by alibama** [comment]
hey dude- thank's for following up - I've got a couple projects that need this.. i'm thinking worst case writing some jscript that finds the elements and rewrites them with alt tags...   given the low quality of my coding standards I think it might be better that way - just feel like the code I'm seeing doesn't really lend itself to being extended as easily as they say it is....

**Comment by fredj** [comment]
What about http://openlayers.org/en/v3.2.1/examples/accessible.html ?

**Comment by alibama** [comment]
I see that - I'm trying to get that on my site, haven't been too successful yet - may be an issue on the drupal end
